### ISSUE TITLE
HTTP ERROR 500

### ISSUE THREAD

**Issue by marcusviniciuspomini** [opensourcepos/opensourcepos#1905]
hey guys, tks for help with this issue.
I'm beginner with these system, cause it i need your help.
I have a local installing of ospos in my UsbWebServer v8.6 
First i created the db(ospos) and executaded the mysql code(database/database.sql)
Second i configurated the connection(	
        'hostname' => !empty(getenv('MYSQL_HOST_NAME')) ? getenv('MYSQL_HOST_NAME') : 'localhost:8080',
	'username' => !empty(getenv('MYSQL_USERNAME')) ? getenv('MYSQL_USERNAME') : 'root',
	'password' => !empty(getenv('MYSQL_PASSWORD')) ? getenv('MYSQL_PASSWORD') : 'usbw',
	'database' => !empty(getenv('MYSQL_DB_NAME')) ? getenv('MYSQL_DB_NAME') : 'ospos',) 
in the(application/config/databse.php)
Then I try acess the system on localhost but when the page is load i receive the error 500 of HTTP and message that page is not working.

What did i do to build on and put the system in work?
I need these just for control the stock and sales on a local server.

**Comment by mckaygerhard** [comment]
error 500 it's miscconfiguration.. maybe due that "rare" thing named usbwebserver that sounds like windo.. 

i noted "localhost:8080" do you have the mysql running in that port? i thinks its your problem.. please check the mysql connection, try running at console: `mysql -u root -h localhost -pusbw -D ospos` if the connection does not happened you must use something better.. try the ISO with all in it

**Comment by marcusviniciuspomini** [comment]
@mckaygerhard  yes, i have mysql running in the 8080 port, also i'm using windows then i can't running the test in the console but I'm sure MySQL runs on port 8080, always i already connected with these.

Today i try config the "ospos" on my webserver of hostinger, I have imported the database.sql to phpmyadmin and put the connect config of server in the database.php file.
And when i try acess the system in your respective link a get a same message of before.

if you can try acess this, the link is vendas9.pe.hu/ospos

**Comment by mckaygerhard** [comment]
1) your mysql connection are not happened, i provided the test to check, if mysql command line does not connect that's the problem, you must tune your mysql database connection, it's clarelly it's the problem, always in that poor OS .. so use linux..

2) about hostinger, current accounts do not have socket and mcrypt enabled so you cannot run ospos in hostinger unless you are a very very older user (those have all the php modules actived)

it's a pain topic here, ospos need so many php modules to run, very few hosting services provide it! digital ocean it's one, but not so many users can paid, configure a docker its also same level of knowledge required.. due that i made a linux OS iso..

**Comment by marcusviniciuspomini** [comment]
@mckaygerhard really that is the problem.
i can solve it using xampp server and not the UsbWebServer who is so limited

**Comment by marcusviniciuspomini** [comment]
tks for help.
:)
